ribs/emitters/_gradient_emitter.py

```diff
@@ -200,7 +200,11 @@ def ask(self):
         """Samples new solutions from a gradient arborescence parameterized by a
         multivariate Gaussian distribution.
 
-        The multivariate Gaussian is parameterized by sigma_g.
+        If measure_gradients is used, the multivariate Gaussian is parameterized
+        by sigma_g, and the arboresecence coefficient is sampled from the
+        multivariate Gaussian, with the objective coefficient being always
+        positive. If measure_gradients is not used, the arboresecence
+        coefficient is just sigma_g itself.
```

Comment:
Please checkout my description on the usage of sigma_g in the docstrings of both ask() and the GradientEmitter class itself.